### DISCUSSION TITLE
v1.5.0 eta?

### DISCUSSION THREAD

**Discussion by ENLinden** [evilsocket/opensnitch#588] [Releases]
Forgive me, but could you provide an ETA on the release of v1.5.0? 

For reasons that would take too long to explain, I can't build from the source and must rely on repo packages (pacman/aur, in this case) because I must be able to ensure all packages are up to date (for security purposes). I'm working on getting the opensnitch-git aur package updated to the rc release, but I have no way of guaranteeing that will occur. So my next best bet is that the final version will be released soon. Do you have good news?

I apologize for my naivete and inexperience. I am using opensnitch as the basis for my anti-distraction measures that I'm now finalizing the implementation of before I start my CompSci degree in May. The sooner I can implement it, the sooner I'll have the time to focus on the rest of my tasks, and one of my scripts relies on the wildcard feature that you implemented at my request.

I'm willing to learn new things and put the effort in to get this to work sooner, but I don't see any way that I can right now. Any news, help, or advice would be greatly appreciated.

**Comment by gustavo-iniguez-goya** [comment]
Hi @ERLindeman ,

I expect to release rc2 soon, probably by the end of this month or earlier. Then after a couple of weeks v1.5.0 can be released.

There's only one big change that I wanted to add, but I can release v1.5.0 and then start working on that change for the v1.6.0.

**Reply by ENLinden** [reply]
Excellent, you are a lifesaver!
I look forward to it. Thank you!
